app/models/motion.rb

```diff
@@ -50,6 +50,16 @@ class Motion < ActiveRecord::Base
   scope :closing_in_24_hours,      -> { where('motions.closing_at > ? AND motions.closing_at <= ?', Time.now, 24.hours.from_now) }
   scope :chronologically, -> { order('created_at asc') }
 
+  scope :closing_soon_not_published, -> {
+     voting
+    .joins("LEFT OUTER JOIN events e ON e.eventable_id = motions.id AND e.eventable_type = 'Motion'")
```

Comment:
The logic here is "give me all of the voting motions where a `motion_closing_soon` event has not been published in the last 2 days".

I'm hypothesizing that there will be use cases where people see 'Oh, hm, that proposal closes in 24 hours, but I think we need longer', and bump it back for a few days, in which case we want to publish another motion_closing_soon event when it's about to close again.

If, however, a motion_closing_soon event was just published say an hour ago, we don't want another one just because the job ran twice.